src/sacn_mock/private/dmx_merger.h

```diff
@@ -0,0 +1,57 @@
+/******************************************************************************
+ * Copyright 2020 ETC Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************
+ * This file is a part of sACN. For more information, go to:
+ * https://github.com/ETCLabs/sACN
+ *****************************************************************************/
+
+#ifndef SACN_MOCK_PRIVATE_DMX_MERGER_H_
+#define SACN_MOCK_PRIVATE_DMX_MERGER_H_
+
+#include "sacn/private/dmx_merger.h"
+
+#ifdef TESTING_CPP_MERGER
```

Comment:
Yeah, this should definitely not be here. The fake module should stand alone, unprotected by #ifdefs, and either be included or excluded from each unit test executable as necessary.